neard/src/config.rs

```diff
@@ -1047,13 +1050,21 @@ pub fn download_genesis(url: &String, path: &PathBuf) {
     });
 }
 
-pub fn load_config(dir: &Path) -> NearConfig {
+pub fn load_config_without_genesis_records(dir: &Path) -> NearConfig {
     let config = Config::from_file(&dir.join(CONFIG_FILENAME));
-    let genesis = if let Some(ref genesis_records_file) = config.genesis_records_file {
-        Genesis::from_files(&dir.join(&config.genesis_file), &dir.join(genesis_records_file))
-    } else {
-        Genesis::from_file(&dir.join(&config.genesis_file))
-    };
+    let genesis_config = GenesisConfig::from_file(&dir.join(&config.genesis_file));
+    let genesis_records_file =
+        if let Some(ref genesis_records_file_path) = config.genesis_records_file {
```

Comment:
As an alternative, to me this version seems simpler: 

```suggestion
        if let Some(genesis_records_file_path) = &config.genesis_records_file {
```